test/info/info-help.test.js

```diff
@@ -17,16 +16,6 @@ describe('should print help for info command', () => {
         expect(stderr).toHaveLength(0);
     });
 
-    it('should respect the --color=false flag', () => {
-        const { stdout, stderr } = runInfo(['help', '--color=false'], __dirname);
-        chalk.enabled = true;
-        chalk.level = 3;
-        const orange = chalk.keyword('orange');
-        expect(stdout).not.toContain(orange(usageText));
-        expect(stdout).toContain(descriptionText);
-        expect(stderr).toHaveLength(0);
-    });
```

Comment:
I think we should not release it, due v4 is not supported core flags, so webpack@4 developers can't disable colors using CLI, some developers disable it only for CI and etc, and they probably don't want to do it in config

Reply:
I think we should be able to disable colors with `colorette` too, https://github.com/jorgebucaran/colorette#optionsenabled

Reply:
Can you clarify?

Reply:
What is the problem, in this PR nothing was changed, only migration

Reply:
We have test with `--color=false`, but we don't have this flag defined in `cli-flag.js`.

Reply:
It was removed in some commit.

Reply:
Let's do migration here and discussion about `--color` in the other PR, now keep logic as is